requirements.txt

```diff
@@ -84,6 +85,7 @@ tomli==2.0.1 ; sys_platform == "darwin"
 torch==2.0.1 ; sys_platform == "darwin"
 tornado==6.1 ; sys_platform == "darwin"
 traitlets==5.1.1 ; sys_platform == "darwin"
+trimesh==3.23.5 ; sys_platform == "darwin"
```

Comment:
this requirements.txt file is automatically generated (using pip tools);  we should add `trimesh<=3.23.5` (with a comment as to the reason) into the jammy-requirements.txt , focal-requirements.txt, and mac-requirements.txt, then run htmlbook/requirements.sh to regenerate

Reply:
done

Reply:
I'm having issues with this where `htmlbook/requirements.sh` does not add the trimesh requirements for python > 3.8. However, executing `cat jammy-requirements.txt | sed -e '/#/d' -e '/^$/d' -e 's/$/; sys_platform == "linux" and python_version > "3.8" /' >> requirements.txt` manually does. I'm not sure what is supposed to happen here